bin/clover/src/pipeline-steps/prettifySocketNames.ts

```diff
@@ -0,0 +1,29 @@
+import { ExpandedPkgSpec } from "../spec/pkgs.ts";
+import { setAnnotationOnSocket } from "../spec/sockets.ts";
+
+export function prettifySocketNames(
+  specs: ExpandedPkgSpec[],
+): ExpandedPkgSpec[] {
+  const newSpecs = [] as ExpandedPkgSpec[];
+
+  for (const spec of specs) {
+    const sockets = spec.schemas[0].variants[0].sockets;
+
+    for (const socket of sockets) {
+      const newName = socket.name
+        // separate any sequence of lowercase letters followed by an uppercase letter
+        .replace(/([a-z])([A-Z])/g, "$1 $2")
+        // Separate any sequence of more than 1 of uppercase letters (acronyms) from the next word
+        .replace(/([A-Z]+)([A-Z][a-z])/g, "$1 $2");
```

Comment:
so this will create `Task Role ARN` as a socket name right?

Reply:
Yes, but aws actually calls arns "Arn" in the sockets that I tested so this wouldn't interact with that. But that's the idea, yeah

Reply:
ah yeah of course not ARN but Arn - I gotcha :)